install/helm/open-match/Chart.yaml

```
@@ -18,8 +18,8 @@ version: 0.0.0-dev
 name: open-match
 dependencies:
   - name: redis
-    version: 16.3.1
-    repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
+    version: 17.7.3
```

Hey Would you like to update it to latest version? Since it is also very near to that you are putting here.
```suggestion
    version: 17.8.0
```